solvestack/glossary/models.py

```diff
@@ -7,3 +7,9 @@ class Term(models.Model):
     id = ShortUUIDField(primary_key=True, editable=False)
     name = models.CharField(max_length=80)
     definition = models.TextField()
+
+class StackNode(models.Model):
+    id = ShortUUIDField(primary_key=True, editable=False)
+    name = models.CharField(max_length=80)
+    definition = models.TextField()
+    similar_tech = models.TextField()
```

Comment:
This is a great opportunity to use a JSONBField or ArrayField so we can at least structure the data as a list of other Technology Nodes with human friendly labels for later functionality